packages/core/aqua-to-js/src/utils.ts

```diff
@@ -27,24 +27,25 @@ import {
   SimpleTypes,
   UnlabeledProductType,
 } from "@fluencelabs/interfaces";
+import { z } from "zod";
 
-export interface PackageJson {
-  name: string;
-  version: string;
-  devDependencies: {
-    ["@fluencelabs/aqua-api"]: string;
-  };
-}
+const packageJsonSchema = z.object({
+  name: z.string(),
+  version: z.string(),
+  devDependencies: z.object({
+    ["@fluencelabs/aqua-api"]: z.string(),
```

Comment:
Some doc on why do you need aqua-api here would be nice

Reply:
Will add